types/emmetio__implicit-tag/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for @emmetio/implicit-tag 1.0
+// Project: https://github.com/emmetio/implicit-tag#readme
+// Definitions by: Robbert Verbruggen <https://github.com/rfverbruggen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module '@emmetio/implicit-tag' {
+    export default function resolveImplicitName(parentName: string): string;
+}
```

Comment:
You shouldn't need to use `declare module` here.

```suggestion
export default declare function resolveImplicitName(parentName: string): string;
```

Reply:
Though it looks like this should probably be the following intead:

```suggestion
declare function resolveImplicitName(parentName: string): string;
export = resolveImplicitName;
```

According to https://unpkg.com/browse/@emmetio/implicit-tag@1.0.0/dist/implicit-tag.cjs.js#L33, the export uses `module.exports = ...` which translates to `export = ...` in TypeScript.

The package has an ES Module output as well, but it's only referenced using the [`"module"` property in package.json](https://unpkg.com/browse/@emmetio/implicit-tag@1.0.0/package.json#L6), which TypeScript doesn't recognize.

Reply:
Implemented your suggestions